File: 1950-sign-of-the-product-of-an-array/sign-of-the-product-of-an-array.py
class Solution(object):
    def arraySign(self, nums):
        for n in nums:
            if n == 0:
                return 0

        neg_count = 0
        for n in nums:
            if n < 0:
                neg_count += 1
        return 1 if neg_count % 2 == 0 else -1

File: 1950-sign-of-the-product-of-an-array/sign-of-the-product-of-an-array.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        int negCount = 0;

        for (int n : nums) {
            if (n == 0) return 0;      
            if (n < 0) negCount++;     
        }

        return (negCount % 2 == 0) ? 1 : -1;
    }
};
